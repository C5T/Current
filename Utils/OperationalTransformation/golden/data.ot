{"A0":{"a":"29505000","o":["def say_hello():\n    print 'Hello, World'\n\nfor i in xrange(5):\n    say_hello()\n"],"t":1478829507704},"A1":{"a":"29505000","o":[43,-19,17],"t":1478829512816},"A2":{"a":"29505000","o":[43,-1,16],"t":1478829513172},"A3":{"a":"29505000","o":[43,-1,15],"t":1478829513468},"A4":{"a":"29505000","o":[43,-1,14],"t":1478829513652},"A5":{"a":"29505000","o":[43,-1,13],"t":1478829514048},"A6":{"a":"29505000","o":[43,-1,12],"t":1478829514254},"A7":{"a":"29505000","o":[28,"П",-12,15],"t":1478829520669},"A8":{"a":"29505000","o":[29,"р",15],"t":1478829520937},"A9":{"a":"29505000","o":[30,"иве",15],"t":1478829521057},"AA":{"a":"29505000","o":[33,"т",15],"t":1478829521153},"AB":{"a":"29505000","o":[34,",",15],"t":1478829521429},"AC":{"a":"29505000","o":[35," ",15],"t":1478829521566},"AD":{"a":"29505000","o":[36,"м",15],"t":1478829522189},"AE":{"a":"29505000","o":[37,"и",15],"t":1478829522359},"AF":{"a":"29505000","o":[38,"р",15],"t":1478829522540},"AG":{"a":"29505000","o":[39,"!",15],"t":1478829522865},"AH":{"a":"29505000","o":[28,"T",-12,15],"t":1478829528226},"AI":{"a":"29505000","o":[29,"e",15],"t":1478829528418},"AJ":{"a":"29505000","o":[30,"s",15],"t":1478829528525},"AK":{"a":"29505000","o":[31,"t",15],"t":1478829528630},"AL":{"a":"29505000","o":[32," ",15],"t":1478829530051},"AM":{"a":"29505000","o":[33,"p",15],"t":1478829530185},"AN":{"a":"29505000","o":[34,"a",15],"t":1478829530308},"AO":{"a":"29505000","o":[35,"s",15],"t":1478829530465},"AP":{"a":"29505000","o":[36,"s",15],"t":1478829530569},"AQ":{"a":"29505000","o":[37,"e",15],"t":1478829530794},"AR":{"a":"29505000","o":[38,"d",15],"t":1478829530937},"AS":{"a":"29505000","o":[39,".",15],"t":1478829531045}}