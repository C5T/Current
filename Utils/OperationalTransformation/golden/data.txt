def say_hello():
    print 'Test passed.'

say_hello()
